### DISCUSSION TITLE
Flash light

### DISCUSSION THREAD

**Discussion by rektosaure** [photobooth-app/photobooth-app#252] [Q&A] [answered]
Hello everyone,

I'm pondering the options available for setting up a flash for a photobooth in combination with a Raspberry Pi v3 camera.

From your experience, which system should I favor? A LED ring? A LED strip? A camera flash? Something else?

How does interfacing with photobooth-app work?

Many thanks for your insights.

**Comment by mgineer85** [comment, marked as answer]
I would go for some indirect but permanent light so the livevpreview works good at any time also. With picsmeras there is no flash solution that is synchronized perfectly currently.

**Reply by rektosaure** [reply]
Thanks for your explanations
